pkgs/applications/networking/instant-messengers/signal-desktop/default.nix

```diff
@@ -114,7 +114,10 @@ stdenv.mkDerivation rec {
 
     # Fix the desktop link
     substituteInPlace $out/share/applications/signal-desktop.desktop \
-      --replace /opt/Signal/signal-desktop $out/bin/signal-desktop
+      --replace /opt/Signal/signal-desktop "$out/bin/signal-desktop --start-in-tray"
+    # Fix end of command - redirect stdout
+    substituteInPlace $out/share/applications/signal-desktop.desktop \
+      --replace "%U" "> /tmp/signal-desktop.log %U"
```

Comment:
This looks like a security vulnerability actually. Signal-desktop should not do this as every process can write to /tmp/signal-desktop.log and replace it with a symlink to something else.

Reply:
I chose "a log file" for any future debugging. We can just pipe it to `/dev/null` to be safe.

Reply:
I was misreading actually what the PR did. I reverted it for now: https://github.com/NixOS/nixpkgs/pull/203456
This is unsafe temp file handling. Piping to /dev/null might work but won't this also make debugging than harder?

Reply:
Previously no such debug info was available. This could be an upgrade or a "feature", but I'm not familiar with a safer log file implementation. Current version would also overwrite the log file on each new launch instead of appending to or creating a new one.

Currently piping to `/dev/null` is a solution to an underlying problem, but we can add a comment for the future, when the problem is fixed on signal's side.